lib/resource.dart

```diff
@@ -19,6 +19,7 @@
 ///     var obj = await resource.openRead()   // Reads as stream of bytes.
 ///                             .transform(utf8.fuse(JSON).decoder)
 ///                             .first;
+@deprecated
```

Comment:
Use full `@Deprecated` syntax pointing folks to help. For many, this will be their first hint the package is deprecated – so the message should be helpful for CI tools and IDEs